lib/ui/painting.dart

```diff
@@ -824,37 +824,73 @@ enum FilterQuality {
 
 /// Styles to use for line endings.
 ///
-/// See [Paint.strokeCap].
+/// See also:
+///
+/// * [Paint.strokeCap] for how this value is used.
+/// * [StrokeJoin] for the different kinds of line segment joins.
 // These enum values must be kept in sync with SkPaint::Cap.
 enum StrokeCap {
   /// Begin and end contours with a flat edge and no extension.
+  ///
+  /// ![Butt Cap Diagram](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/butt_cap.png)
```

Comment:
"Butt Cap Diagram" is worthless alternative text. (It's a caption, not alternative text.) See https://html.spec.whatwg.org/multipage/images.html#alt